### ISSUE TITLE
Warning spam

### ISSUE THREAD

**Issue by lukedbx** [ArmaLife/Altis#84]
Wondered if you know whats causing this. Basically local rpt gets spammed with hundreds of 
 13:42:48 WARNING: Function 'name' - cop_1 has no unit
13:42:48  - network id 2:3
13:42:48  - person dave
13:42:48  - dead

**Comment by tkcjesse** [comment]
Can you pastebin your whole rpt out please.

**Comment by Heavybob** [comment]
RPT, you running main build or a modded mission.sqm?

**Comment by Heavybob** [comment]
*Close if no rpt logs*

**Comment by danielstuart14** [comment]
Not related to life mod

**Comment by tkcjesse** [comment]
Um, I'm actually getting the same error: 
20:23:36 WARNING: Function 'name' - cop_1 has no unit
20:23:36  - network id 4:40
20:23:36  - person [BW]fuzionz.
20:23:36  - dead

-- This is client side RPT.

**Comment by tkcjesse** [comment]
Think it has something to do with playertags... when users die...? Will report back with more info shortly..

**Comment by tkcjesse** [comment]
Regarding: https://dev.withsix.com/issues/13861

**Comment by lukedbx** [comment]
Oh sorry I totally forgot I posted this as it seems to have stopped now and I didn't change anything. However when I was reading up on it I found this on the bistudio site, not sure if you saw it.

WARNING: Function 'name' - So-and-so is dead
Do not call name on a dead object. if (alive _obj) then {name _obj};

https://community.bistudio.com/wiki/arma.RPT

**Comment by tkcjesse** [comment]
Yeah. I'm currently looking into it kind of.. Seems to be a player tag issue when people die.

**Comment by lukedbx** [comment]
Also RPT if you still need it - http://pastebin.com/f3HA2y6N

**Comment by tkcjesse** [comment]
Yeah. Same as mine. Thanks though!

**Comment by tkcjesse** [comment]
Check @lukedbx

https://github.com/tkcjesse/Altis/commit/7ff46c2f3a5bf4fefdd904fb7deab6be32502983

**Comment by lukedbx** [comment]
Thanks, will try this out and let you know.

**Comment by tkcjesse** [comment]
Yep. No problem.  Let me know.

**Comment by lukedbx** [comment]
Still have the same problem unfortunately, and now im also getting

**Comment by tkcjesse** [comment]
Can you verify that you have this change too please. https://github.com/ArmaLife/Altis/commit/193343ffe2aae148719ea8b4ddd6ffde6fbb3573#diff-d409e875a3bfc749662fb82a0adf9988

**Comment by lukedbx** [comment]
Yea, have that.

**Comment by tkcjesse** [comment]
Do you have names above npcs lke the player nametags do? And on the gang pallets?

**Comment by lukedbx** [comment]
Yea

**Comment by tkcjesse** [comment]
So other than those 3 fixes I just put above. I removed that from my NPCs and all that spam was completely removed. Because I also got spam from the pallets. You could try it... But that's up to you. Those fixes fixed my stuff...

**Comment by lukedbx** [comment]
Yea I will do, thanks for your help.

**Comment by tkcjesse** [comment]
Of course.

**Comment by Jawshy** [comment]
Could this issue be reopened as these errors and new ones appear to still be present in the local/client.rpt log: 

```
21:10:51 Bad conversion: scalar
```
```
21:22:14 WARNING: Function 'name' - MATTY is dead
```
```
21:22:16 WARNING: Function 'name' - civ_2 has no unit
21:22:16  - network id 2:1216
21:22:16  - person Athanasiadis
21:22:16  - dead
```

What I did was kill my test subject and then revive him. We then switched sides and did the same thing. Also a player that is revived losses their player tags (above their head). 

Link to [.zip](http://puu.sh/mRLfK/28a0820033.zip) containing both the full client/local.rpt and the server.rpt

Game version: 1.54.133741
Mission version: v4.3r4 - latest commit: c98c6409d2d86a0e58be3d381feb388a7b35b87f
